tests/console/check_system_info.pm

```diff
@@ -94,9 +94,10 @@ sub check_nodejs_common {
     zypper_call('in nodejs-common | tee /tmp/install_nodejs.log');
 
     assert_script_run('grep -E "Installing: nodejs16" /tmp/install_nodejs.log');
-    my $ret_s1 = script_output(q(zypper se -i nodejs | sed -n '/nodejs16 /p' | awk '{print $1}'));
-    my $ret_s2 = script_output(q(zypper se -i nodejs | sed -n '/nodejs-common/p' | awk '{print $1}'));
-    die "Expected package is not installed" if ('i' ne "$ret_s1" || 'i+' ne "$ret_s2");
+    script_run('zypper se -i nodejs');
+    my $ret_s1 = script_output(q(zypper se -i nodejs|grep package|grep nodejs16|awk -F\| '{print $1}'));
+    my $ret_s2 = script_output(q(zypper se -i nodejs|grep package|grep nodejs-common|awk -F\| '{print $1}'));
+    die "Expected package is not installed. ret_s1=$ret_s1' and ret_s2=$ret_s2'" if ('i' ne "$ret_s1" || 'i+' ne "$ret_s2");
```

Comment:
@hjluo why adding ' and ret_s2=$ret_s2' here?

Reply:
I mean 
ret_s1=$ret_s1 and ret_s2=$ret_s2 
should be fine

Reply:
> @hjluo why adding ' and ret_s2=$ret_s2' here?

this is just printing what's wrong when our check failed.

Reply:
if there's space it's hard to identify with eyes. thanks